include/defines.h

```diff
@@ -54,7 +54,11 @@
 
 #ifdef VERSION_EU
 #define COURSE_TIMER_ITER 0.020041665999999999    // 1 / 50
-#define COURSE_TIMER_ITER_f 0.020041665999999999f // 1 / 50
+    #ifdef AVOID_UB
+    #define COURSE_TIMER_ITER_f 0.020041665999999999f // 1 / 50
+    #else
+    #define COURSE_TIMER_ITER_f 0.01666666f //! 1 / 60 - EU has course timer untouched for floats
+    #endif
```

Comment:
Could you please add a short note explaining why there's an _f version?

Reply:
Sure, could be for matching reason, but if not I'll just rename it